csharp/test/Ice/timeout/Server.cs

```diff
@@ -17,11 +17,11 @@ public override async Task RunAsync(string[] args)
             properties["Ice.Warn.Connections"] = "0";
 
             // The client sends large messages to cause the transport buffers to fill up.
-            properties["Ice.MessageSizeMax"] = "20000";
+            properties["Ice.MessageSizeMax"] = "20MB";
```

Comment:
Should this be `20KB`?

```suggestion
            properties["Ice.MessageSizeMax"] = "20KB";
```

Reply:
MessageSizeMax used to be KB, so I think 20 MB is correct here

https://doc.zeroc.com/ice/3.7/property-reference/miscellaneous-ice-properties#id-.MiscellaneousIce.*Propertiesv3.7-Ice.MessageSizeMax

Reply:
Ah, I was comparing with `Ice.TCP.RcvSize`, which is I guess in bytes?

Reply:
yes